Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby:2.4
+
+RUN apt-get update \
+  && apt-get install -y \
+    git \
+    make
+
+COPY . /gh-pages
```

Comment:
Due to how `Jekyll.sanitized_path` works, let's put this in a subdirectory 2 levels deep, e.g. `/gh/pages` or `/src/gh-pages`. Otherwise, if you have a file called `gh-pages.html`, you'll get an error. It's confusing so let's tackle that before merging.

Reply:
cbe4cdf54238210673752bdecf44ebc1e0c08d00